### PR TITLE
Merging and tweaking

### PR DESCRIPTION
**Pull request by cloveras**
* Merged as in https://github.com/vippsas/vipps-ecom-api/issues/116
* Removed a few "DTO" suffixes
* Removed "get" from some schemas
* Minor improvements for less WTFs

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -1562,18 +1490,19 @@ components:
             for more information.
         timeStamp:
           type: string
-          description: Timestamp in ISO-8601 representing when the order was captured.
+          description: Timestamp in ISO-8601 representing when the order was cancelled.
```
Comment (tht13) [inline docs/swagger.yaml:1493]: make description generic for all three events "when operation was processed on the order"?

Comment (cloveras) [reply docs/swagger.yaml:1493]: Ah! Thanks.

**Comment by tht13** [comment]
Found some references that need updating
https://github.com/vippsas/vipps-ecom-api/blob/0278f9b893cbacc3c61ba5b92c197239d7beaf89/docs/swagger.yaml#L526


https://github.com/vippsas/vipps-ecom-api/blob/0278f9b893cbacc3c61ba5b92c197239d7beaf89/docs/swagger.yaml#L644


https://github.com/vippsas/vipps-ecom-api/blob/0278f9b893cbacc3c61ba5b92c197239d7beaf89/docs/swagger.yaml#L769

**Comment by tht13** [comment]
you're too quick.... 🛥️

**Comment by cloveras** [comment]
Fixed ⚡️

**Pull request merged**